modules/test-e2e/src/test/scala/org/knora/webapi/slice/resources/api/ValuesEndpointsE2ESpec.scala

```diff
@@ -1566,6 +1566,44 @@ object ValuesEndpointsE2ESpec extends E2EZSpec { self =>
         endEra == dateValueHasStartEra,
       )
     },
+    test("create a text value with standoff containing a footnote with apostrophe (encode apostrophe correctly)") {
+      val resourceIri: IRI = AThing.iri
+      val textValueAsXml: String =
+        """|<?xml version="1.0" encoding="UTF-8"?>
+           |<text>Text <footnote content="Apostrophe start ' end"/> end text</text>
+           |""".stripMargin
+
+      val propertyIri: SmartIri = "http://0.0.0.0:3333/ontology/0001/anything/v2#hasText".toSmartIri
+
+      for {
+        maybeResourceLastModDate: Option[Instant] <- getResourceLastModificationDate(resourceIri)
+
+        jsonLd = createTextValueWithStandoffRequest(
+                   resourceIri = resourceIri,
+                   textValueAsXml = textValueAsXml,
+                   mappingIri = standardMappingIri,
+                 )
+
+        responseJsonDoc <- TestApiClient.postJsonLdDocument(uri"/v2/values", jsonLd, anythingUser1).flatMap(_.assert200)
+        valueIri        <- ZIO.fromEither(responseJsonDoc.body.getRequiredString(JsonLDKeywords.ID))
+        valueType       <- ZIO.fromEither(responseJsonDoc.body.getRequiredString(JsonLDKeywords.TYPE))
+
+        savedValue <- getValue(
+                        resourceIri = resourceIri,
+                        maybePreviousLastModDate = maybeResourceLastModDate,
+                        propertyIriForGravsearch = propertyIri,
+                        propertyIriInResult = propertyIri,
+                        expectedValueIri = valueIri,
+                      )
+
+        savedTextValueAsXml <- ZIO.fromEither(
+                                 savedValue.getRequiredString(KA.TextValueAsXml),
+                               )
+      } yield {
+        // The apostrophe should be preserved as-is, not escaped as "\&apos;"
+        assertTrue(savedTextValueAsXml.contains("Apostrophe start &apos; end"))
```

Comment:
question: It the comment right or the assert?

They are contradicting each other.